docs/user-stories.md

```diff
@@ -0,0 +1,9 @@
+## Glottie Chat User Stories
+
+1. As a researcher whose native language is not English, I want to communicate with peer colleagues in a chat-room where all messages are translated automatically, so that I don't have to fight with English grammar and correct word usage.
+2. As a chat service user, I want to obtain a unique account, so that my personal identity couldn't be forged.
+3. As an authenticated user, I need to be able to create new chat-rooms dedicated to cerain topics and invite other fellow researchers to join them.
+4. As a chat participant, I want to preview the content of each of my messages and make sure the system makes an adequate semantic interpretation of it, so that I could rest assured the intended meaning will come across.
```

Comment:
What if an interpretation won't satisfy the author of a message? Will they be writing messages until a satisfiable version?

Reply:
I'm afraid they will, otherwise there will be no guarantee of meaning coherence that is essential for effective communication. Do you think this approach might look pedantic and off-putting?

Reply:
I can suggest adding a possibility to edit the translated message as well, a lot of researchers can not have a desire to translate a full text, but still are able to read the translation and add some minor changes. (e.g. when theorem name was translated too literally, sure they will be able to provide its correct commonly-used name in English)

Reply:
Actually, I thought the meaning of one's input message should be restated using the same language, not necessarily English. The actual translation to other languages shall be generated as a second phase. E.g. 1. Russian -> Russian (in other words, with internal representation in a formal semantic language) 2. Formal semantic language -> target language (eg. German).

Reply:
Got it 👍